libraries/app/include/graphene/app/database_api.hpp

```diff
@@ -596,6 +596,11 @@ class database_api
       /// @brief Get a hexdump of the serialized binary form of a transaction
       std::string get_transaction_hex(const signed_transaction& trx)const;
 
+      /// @brief Get a hexdump of the serialized binary form of a
+      /// signatures-stripped transaction
+      std::string
+      get_transaction_hex_without_sig(const signed_transaction &trx) const;
+
```

Comment:
please put `std::string` and `get_transaction_hex_without_sig(const signed_transaction &trx) const;` in the same line. thanks!

Reply:
i am wondering also what will be happening if a transaction with signatures came here. what will be the output ? maybe add a test case for that?

Reply:
+1 please add a test case.

Reply:
Thanks Alfredo and Peter, fixed